src/utils/yourServices.ts

```diff
@@ -81,11 +82,13 @@ export const getAllowedListServices = async (
 
 export const formatService = (service: Service): Service => {
   const readable_format_date = prettifyDate(service.last_accessed);
+  const isHMRC = hmrcClientIds.includes(service.client_id);
   return {
     client_id: service.client_id,
     count_successful_logins: service.count_successful_logins,
     last_accessed: service.last_accessed,
     last_accessed_readable_format: readable_format_date,
+    isHMRC: isHMRC,
   };
 };
 
```

Comment:
Should we have included this in a unit test?  Does this change reduce the Sonarqube coverage score?